i18n/zh/docusaurus-plugin-content-docs/version-2.0-2.4.json

```diff
@@ -260,11 +260,11 @@
     "description": "The label for category Provisioning Storage Examples in sidebar tutorialSidebar"
   },
   "sidebar.tutorialSidebar.category.Manage Projects": {
-    "message": "Manage Projects",
+    "message": "Project Administration",
```

Comment:
To match title, and to match the label in new user guides, "Cluster Administration"